### PR TITLE
[11.0][MIG] document page quality manual

### PR DESCRIPTION
**Pull request by jalzaga**
Module document_page_quality_manual Migration to version 11.0 #200

### CODE REVIEW

File: document_page_quality_manual/migrations/7.0.1.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+##############################################################################
```
Comment (ivantodorovich) [inline document_page_quality_manual/migrations/7.0.1.0/pre-migration.py]: Remove old migration scripts

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Comment by ivantodorovich** [comment]
Minor comment, otherwise LGTM
Since it depends on `mgmtsystem_manual`, we need to wait for https://github.com/OCA/management-system/pull/215 to get merged

**Comment by jalzaga** [comment]
@ivantodorovich @max3903 Can you please check why te runbot error? I think the module is ready

**Comment by jalzaga** [comment]
@ivantodorovich @max3903 Could you pkease let me know what the problem is regarding runbot and travis?

File: document_page_quality_manual/README.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+=====
+
+To enter your quality manual, you need to:
+
+* go to Management Systems > Documentation > Manuals
+* Create a new page and select the Quality Manual category.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/118/10.0
```
Comment (max3903) [inline document_page_quality_manual/README.rst]: /11.0

File: document_page_quality_manual/README.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* Report any issue or ideas to https://github.com/OCA/management-system/issues
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/management-system/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/management-system/issues/new?body=module:%20document_page_quality_manual%0Aversion:%2010.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (max3903) [inline document_page_quality_manual/README.rst]: 11.0

**Review by max3903 — CHANGES_REQUESTED** [review]
Minor changes. LGTM

**Comment by jalzaga** [comment]
@max3903 Could you please review this PR? I think it is OK

**Review by ivantodorovich — APPROVED** [review]

**Review by max3903 — APPROVED** [review]

**Pull request merged**